org.gnome.gnome-mahjongg.yaml

```diff
@@ -0,0 +1,28 @@
+app-id: org.gnome.gnome-mahjongg
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '3.30'
+command: gnome-mahjongg
+rename-desktop-file: gnome-mahjongg.desktop
```

Comment:
Ideally these renames should be also done upstream.

Reply:
@barthalion Everything was done here https://gitlab.gnome.org/GNOME/gnome-mahjongg/merge_requests/5